Introductory Tests/traj_opt.ipynb

```diff
@@ -2,36 +2,39 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #58.        population_size = 10 #10
</pre><p>you might wanna add an assert &gt;= 7 somewhere (or comment)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rasmusmarak/67P-Swarm/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>